exercises/space-age/description.md

```diff
@@ -1,8 +1,8 @@
 Given an age in seconds, calculate how old someone would be on:
 
-   - Earth: orbital period 365.25 Earth days, or 31557600 seconds
    - Mercury: orbital period 0.2408467 Earth years
    - Venus: orbital period 0.61519726 Earth years
+   - Earth: orbital period 1.0 Earth years, 365.25 Earth days, or 31,557,600 seconds
```

Comment:
Not a fan of using comma as separator because it can look weird to people who aren't from English-speaking countries. The international standard would be a non-breaking space.

(I don't mind it that much but thought it's worth bringing up)

Reply:
Really two changes in one blow there.  Should have been just a "put earth in it's proper order" and not a format change at the same time.

It was fine without spaces, without comma's.  I say we leave it as it was, but with a new order?  It also made it really easy for students to copy and paste the value, without any worry, just like the other numbers presented here.

Let's keep it "easy" as well as "simple".

```suggestion
   - Earth: orbital period 1.0 Earth years, 365.25 Earth days, or 31557600 seconds
```